src/bootstrap_handler.rs

```diff
@@ -129,8 +121,27 @@ impl BootstrapHandler {
 
     fn write_bootstrap_file(&mut self, mut contacts: Contacts) -> io::Result<()> {
         contacts = contacts.clone().into_iter().unique().collect();
-        self.file_path = BootstrapHandler::get_file_path();
-        let mut file = try!(File::create(&self.file_path));
+        let _ =  fs::create_dir_all(self.file_path.parent().unwrap());
```

Comment:
I'm not sure.  Obviously the best outcome is to have no `unwrap`s left in the production code, but at the very least this PR shouldn't be adding more!  If it's quick to just remove the pre-existing ones I'd probably just do them too.  Probably best to just clear your decision with @Viv-Rajkumar.